### DISCUSSION TITLE
Activity Log Question

### DISCUSSION THREAD

**Discussion by laverdealexander** [Laravel-Backpack/community-forum#748] [Q&A (Help)]
I get this error when i try to use Activity Log addon. Filter is a Backpack PRO feature. Please purchase and install the Backpack\PRO addon from backpackforlaravel.com. I need to purchase any package to use this? Thanks

**Comment by pxpm** [comment]
Hey there @laverdealexander 👋  Thanks for the report! 🙏 

Indeed `Filters` are a PRO feature, but they shouldn't prevent you from using the package if you don't have the PRO version, I totally agree.

I've just submitted https://github.com/Laravel-Backpack/activity-log/pull/33 that addresses this issue. I've asked @promatik to review and merge it if everything looks good to him. 

Cheers

**Reply by laverdealexander** [reply]
Thanks for help! Issue solved super fast! Great work! Thanks for all this tools!

**Reply by pxpm** [reply]
Sure, glad I can help 🙏 

Just merged and tagged 2.0.1, it should be available with a `composer update` 👍 

Cheers
